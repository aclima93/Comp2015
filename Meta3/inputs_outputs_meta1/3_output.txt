Line 2, col 1: syntax error: and
