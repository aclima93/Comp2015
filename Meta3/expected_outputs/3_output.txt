Line 14, col 36: Symbol c not defined
